src/docs/reference/core/Apis.GraphQL.Abstractions/README.md

```diff
@@ -176,6 +178,120 @@ Shown in the example above, we have an autoroutePart argument, this is registere
 
 Done.
 
+### Extending Content Type Query Filters
+
+In the previous section, we demonstrated how to create custom filters for more complex requirements. However, if you need to implement filtering based on your index atomic values, the `WhereInputObjectGraphType` and `IIndexAliasProvider` are what you need.
```

Comment:
What do you mean by "index atomic values"?

Reply:
It was mentioned to clarify that you should have atomic values in index for filtering - e.g. to prevent someone from filtering on "Content" field of Document table :) but i removed it